frontend/src/TransitPredictionsPage.js

```diff
@@ -24,23 +24,46 @@ export default function TransitPredictionsPage() {
     getSCEtaPredictions();
   }, []);
 
+  function encode(stopName) {
+    return stopName.replace(/\s+/g, '-')
```

Comment:
add a comment above on what this replace is doing here and an example like
```js
// this replaces all non alpha numeric chars in the stop name with dashes.
// it also replaces the character "&" with "and", as well as convert all the
// characters in the stop to lowercase.
// for example "Santa Clara & 6th" becomes "santa-clara-and-6th"
```

Reply:
added + resolved